test/xkeyboard-config-test.py.in

```diff
@@ -265,6 +265,10 @@ def main(args):
     parser.add_argument('--verbose', '-v', default=False, action="store_true")
     parser.add_argument('--keymap-output-dir', default=None, type=str,
                         help='Directory to print compiled keymaps to')
+    parser.add_argument('--layout', default=None, type=str)
```

Comment:
Maybe a help "Only test this specific layout" etc?

Reply:
help added, thanks